server/manager/src/test/java/org/apache/accumulo/manager/tableOps/split/UpdateTabletsTest.java

```diff
@@ -261,6 +261,7 @@ public void testManyColumns() throws Exception {
     EasyMock.expect(tabletMeta.getHostingRequested()).andReturn(true).atLeastOnce();
     EasyMock.expect(tabletMeta.getSuspend()).andReturn(suspendingTServer).atLeastOnce();
     EasyMock.expect(tabletMeta.getLast()).andReturn(lastLocation).atLeastOnce();
+    EasyMock.expect(tabletMeta.getUnSplittable()).andReturn(null).atLeastOnce();
```

Comment:
Would be good to test the case where its present to ensure its deleted.

```suggestion
    UnSplittableMetadata usm = UnSplittableMetadata.toUnSplittable(origExtent, 1000, 1001, 1002, tabletFiles.keySet());
    EasyMock.expect(tabletMeta.getUnSplittable()).andReturn(usm).atLeastOnce();
```

Then later in the code where it checks what methods were called on the mutator could do the following.

```java
EasyMock.expect(tablet3Mutator.deleteUnSplittable()).andReturn(tablet3Mutator);
```